src/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Drawer } from '../index.js'
+import { Story } from '@storybook/react'
+import styled from 'styled-components'
+
+export default {
+  title: 'Drawer',
+  component: Drawer,
+}
+
+export const Simple: Story = () => {
```

Comment:
since this is HII specific maybe we would like distinguish between HII and default rather than simple is hii or we do not want? 

also if you would put into the Components/Drawer title it would be wrapped in components, this would allow to create some page examples as well or some hooks.

Reply:
Simple simply refers to passing minimal properties here. I'm not really considering this HII specific at this point, trying to be generic, and just thinking of this as a starting point